tensorflow-sources.file

```diff
@@ -12,9 +12,10 @@
 BuildRequires: bazel swig java-env git
 Requires: eigen protobuf python py2-numpy py2-enum34 py2-mock py2-wheel py2-keras-applications py2-keras-preprocessing py2-setuptools py2-future
 
-%define tag         c0067ec0da9bcdbc258e447bb4477412ea5e4136
+#%define tag         1a9d6b589044172410202ebca2feb44e5efa1734
```

Comment:
Needs to be changed to cms-externals once ported from https://github.com/riga/tensorflow/tree/cms/v2.1.0.

Reply:
I'll take care of the "formalities" once the test have passed.